package.json

```diff
@@ -14,9 +14,11 @@
   },
   "dependencies": {
     "html-webpack-plugin": "^2.28.0",
+    "leaflet": "^1.0.3",
```

Comment:
I'm not sure about this, I would like to ask the tutors, when should we use dependencies or devDependencies? I always get confused: I'm guessing webpack will bundle only dependencies inside a production environment, but we generally install only what we need. Don't we need every dependency? Except maybe webpack-dev-server. @kordianbruck @sacdallago